Specs/Scene/Cesium3DTileFeatureTableResourcesSpec.js

```diff
@@ -0,0 +1,44 @@
+/*global defineSuite*/
+defineSuite([
+    'Core/ComponentDatatype',
+    'Scene/Cesium3DTileFeatureTableResources'
```

Comment:
`Scene/Cesium3DTileFeatureTableResources` should be included first so that it appears correctly in the jasmine specs list.